rotkehlchen/data/importer.py

```diff
@@ -406,18 +412,41 @@ def _import_cryptocom_associated_entries(self, data: Any, tx_kind: str) -> None:
         investments_withdrawals: Dict[str, List[Any]] = defaultdict(list)
         debited_row = None
         credited_row = None
+        expects_debited = False
+        credited_timestamp = None
         for row in data:
+            # If we don't have the corresponding debited entry ignore them
+            # and warn the user
+            if (
+                expects_debited is True and
+                row['Transaction Kind'] != f'{tx_kind}_debited'
+            ):
+                self.db.msg_aggregator.add_warning(
+                    f'Error during cryptocom CSV import consumption. Found {tx_kind}_credited '
+                    f'but no amount debited after at date {row["Timestamp (UTC)"]}',
+                )
+                # Pop the last debited event as it's invalid
```

Comment:
Why is it invalid? Write it in the comment.

Reply:
We always assume to be at least one debited event and one credited event. If this doesn't happen our logic breaks. And this should always happens but the output from crypto.com does strange things sometimes

Reply:
As I said ... write it in a condensed way in the comment in the code. Nobody will remember what you wrote here in the PR.